common/src/main/scala/com/gu/config/Configuration.scala

```diff
@@ -19,41 +17,18 @@ object Configuration extends LazyLogging {
     .getOrElse("TRUE").toBoolean)
     .getOrElse(true) //Should we load config from S3
 
-  val stage = Stage(Option(System.getenv("GU_SUPPORT_WORKERS_STAGE")).getOrElse(Stages.DEV))
+  val stage = Stage.fromString(System.getenv("GU_SUPPORT_WORKERS_STAGE")).getOrElse(Stages.DEV)
```

Comment:
It's fine because if the environment variable is null then the match fails in .fromString and we use the default.